### PR TITLE
Mekanism Power Pass

### PR DESCRIPTION
**Pull request by MuteTiefling**
Re-introduces Passive energy from the Fusion reactor, though at a still greatly reduced rate over default. An Injection Rate of 99 settles in at about 116k FE/t without water cooling. With water cooling, it drops to around 35k FE/t.

Also backed out the steam changes a tiny bit as it has no impact on total energy production, just on the number of turbines needed to get that energy and your ability to cool fission reactors. Should be slightly easier now. 

#1662

### CODE REVIEW

File: config/Mekanism/generators.toml [hunk 1]
```diff
@@ -82,7 +82,7 @@
 	[generators.fusion_reactor]
 		#The fraction of the heat dissipated from the case that is converted to Joules.
 		#Range: 0.0 ~ 1.0
-		thermocoupleEfficiency = 0
+		thermocoupleEfficiency = 0.007
```
Comment (NillerMedDild) [inline config/Mekanism/generators.toml:85]: Would a value of 0.014 translate to roughly double the values in the PR description?

Comment (MuteTiefling) [reply config/Mekanism/generators.toml:85]: ... maybe?

Comment (MuteTiefling) [reply config/Mekanism/generators.toml:85]: I'll test it here and let you know

Comment (MuteTiefling) [reply config/Mekanism/generators.toml:85]: Short answer, no.

Long answer.. it's going to take a few minutes for the heat to stabilize to get a good read.

Comment (MuteTiefling) [reply config/Mekanism/generators.toml:85]: Well, ok. I'm a dirty liar. It looked like it was going to grow faster than linearly after the initial adjustment, but it has stabilized off to pretty much exactly double the air cooled value with an efficiency of 0.014.

Comment (NillerMedDild) [reply config/Mekanism/generators.toml:85]: 😄 I would like the value to be `0.014` then, seems fair considering the injection rate I think. If I'm understanding correctly :P

Comment (MuteTiefling) [reply config/Mekanism/generators.toml]: And up she goes

**Review by NillerMedDild — COMMENTED** [review]
Thanks!

**Pull request merged**